packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -686,6 +686,102 @@ inline void fromRawValue(
   react_native_expect(false);
 }
 
+// [macOS [visionOS]
+inline void fromRawValue(
+    const PropsParserContext& context,
+    const RawValue& value,
+    Cursor& result) {
+  result = Cursor::Auto;
+  react_native_expect(value.hasType<std::string>());
+  if (!value.hasType<std::string>()) {
+    return;
+  }
+  auto stringValue = std::string_view{(std::string)value};
```

Comment:
Got here from https://github.com/facebook/react-native/pull/43218. This is definitely not what you want - see https://github.com/facebook/react-native/pull/43218#discussion_r1505541302.

Reply:
Thanks, I appreciate it. Reverting the change, and I'll keep the upstream PR up a little longer in case we find a better solution.